tests/Nfield.Quota.Tests/QuotaFrameValidatorTests.cs

```diff
@@ -700,6 +700,171 @@ public void Frame_CannotHaveMoreLevelsInTheFrameThanUnderTheReferencedVariableDe
             Assert.That(result.Errors.Single().ErrorMessage,
                 Is.EqualTo(expectedErrorMessage));
         }
+        [Test]
+        public void Frame_CannotHaveLevelTargetWithNegativeValue()
+        {
+            var varId = Guid.NewGuid();
+            var level1Id = Guid.NewGuid();
+            var level2Id = Guid.NewGuid();
+            var level3Id = Guid.NewGuid();
+            const string level3Name = "lvl3";
+            const int invalidTarget = -10;
+
+            var expectedErrorMessage = string.Format(CultureInfo.InvariantCulture,
+                "Target invalid. All Targets must be of a positive value. Frame level id '{0}' with Name '{1}' has a negative Target '{2}'",
+                level3Id,level3Name,invalidTarget);
+
+            var quotaFrame = new QuotaFrame()
```

Comment:
Please use the builder to create (valid) trees :) The reason some other tests create a tree using this syntax is because they need to create invalid trees which are not possible by using our builder :)